kb/prometheus-grafana-setup-cloud.md

```diff
@@ -0,0 +1,252 @@
+---
+slug: prometheus-grafana-setup-cloud
+title: Set up Grafana with Temporal Cloud Observability to view metrics
+tags:
+  - kb-article
+date: 2022-12-02T00:00:00Z
+---
+
+Temporal Cloud and SDKs emit metrics that can be used to monitor performance and troubleshoot errors.
+
+While Temporal Cloud emits metrics through a Prometheus HTTP API endpoint, the open-source SDKs require you to set up a Prometheus scrape endpoint to collect and aggregate the Worker and Client metrics.
+
+This article describes how to set up your Temporal Cloud and SDK metrics, and use them as data sources in Grafana.
+
+The process for setting up Observability includes the following steps:
```

Comment:
Again, this is init-capped because the UI section is labeled this way; but yeah the process that follows is for observability in general and not just run the cloud ui